crates/ruff_python_formatter/src/string/any.rs

```diff
@@ -6,14 +6,15 @@ use ruff_python_ast::{
     self as ast, AnyNodeRef, Expr, ExprBytesLiteral, ExprFString, ExprStringLiteral, ExpressionRef,
     StringLiteral,
 };
+use ruff_python_parser::{StringKind, StringPart};
```

Comment:
We should move `StringKind` to `ruff_python_ast` if it is something that other crates should depend on. Ideally, the formatter only depends on `ruff_python_ast` and gets passed an AST, instead of calling parsing itself (because it doesn't really care about how the file was parsed, all it needs is the ast)